batchflow/models/tf/mobilenet.py

```diff
@@ -189,28 +230,41 @@ def body(cls, inputs, name='body', **kwargs):
                 for k in range(repeats):
                     if k > 0:
                         block['strides'] = 1
-                    x = cls.block(x, **block, residual=k > 0, name='block-%d' % i, **kwargs)
+                    residual = block.pop('residual', False) or k > 0
+                    block = {**kwargs, **block}
+                    x = cls.block(x, **block, residual=residual, name='block-%d' % i)
```

Comment:
What about moving `repeat` logic inside block, with `repeat` being one of the `kwargs`? Same goes for `residual`. That would allow to use `body` very similar to the `MobileNet` one.